src/commands/init/init.py

```diff
@@ -1,53 +1,84 @@
 import shutil
 from pathlib import Path
+from typing import Any
 
-from colorama import Fore
 from git.repo import Repo
 
 from src.utils import log_color_provider
 from src.utils.config.project import Project, ProjectConfig
 
 
-def init(script_root: str):
+def init(args: Any, script_root: str):
     """
     Creates init protostar project
     """
-    project_name = input(f"{Fore.CYAN}Project name: ")
+    if args.existing:
+        project_creator = OnlyConfigCreator(script_root)
+    else:
+        project_creator = ProjectCreator(script_root)
 
-    project_root = Path() / project_name
-    copy_template(script_root, "default", project_root)
-    project = Project(project_root=project_root)
+    project_creator.create()
 
-    def colored_input(message: str):
+
+class ProjectCreator:
+    def __init__(self, script_root: str):
+        self.script_root = script_root
+        self.config = ProjectConfig()
+
+    @staticmethod
+    def request_input(message: str):
         return input(
             f"{log_color_provider.get_color('CYAN')}{message}:{log_color_provider.get_color('RESET')} "
         )
 
-    project_description = colored_input("Project description")
-    author = colored_input("Author")
-    version = colored_input("Version")
-    project_license = colored_input("License")
-    lib_dir = colored_input("Libraries directory name (optional)")
-
-    lib_pth = Path(project_root, lib_dir)
+    def create(self):
+        self.interactive_input()
+        self.project_creation()
 
-    if lib_dir and not lib_pth.is_dir():
-        lib_pth.mkdir(parents=True)
-
-    project.write_config(
-        ProjectConfig(
+    def interactive_input(self):
+        project_name = ProjectCreator.request_input("Project name: ")
```

Comment:
Currently, Protostar crashes when a user provides empty `project_name`. Can you fix it?

Reply:
Good catch